crates/viewer/re_space_view_spatial/src/view_3d.rs

```diff
@@ -443,3 +446,84 @@ impl SpaceViewClass for SpatialSpaceView3D {
         self.view_3d(ctx, ui, state, query, system_output)
     }
 }
+
+fn view_property_ui_grid3d(
```

Comment:
worth a quick comment to state why is it that `view_property_ui()` wasn't good enough here